editor/project_manager.cpp

```diff
@@ -612,7 +612,13 @@ void ProjectManager::_open_selected_projects_check_warnings() {
 }
 
 void ProjectManager::_open_selected_projects_check_recovery_mode() {
-	ProjectList::Item project = project_list->get_selected_projects()[0];
+	Vector<ProjectList::Item> selected_projects = project_list->get_selected_projects();
+
+	if (selected_projects.is_empty()) {
+		return;
+	}
+
+	ProjectList::Item project = selected_projects[0];
```

Comment:
I forgot to make this a `cref` to avoid an unnecessary copy; since you're modifying this, can you add it please?
```suggestion
	const ProjectList::Item& project = selected_projects[0];
```

Reply:
Done!